backend/app/role_assignments/global_/router.py

```diff
@@ -34,6 +36,39 @@ def list_assignments(
     return RoleAssignmentService(db).list_assignments(user_id=user_id, role_id=role_id)
 
 
+@router.post(
+    "/become_admin",
+)
+def become_admin(
+    request: BecomeAdmin,
+    db: Session = Depends(get_db_session),
+    user: User = Depends(get_authenticated_user),
+) -> bool:
+    # TODO Check this in a resolver?
```

Comment:
Either fix Todo, or remove it. I rather have a github issue than a todo in the code :)